modules/datadog-monitors.nix

```diff
@@ -48,14 +48,15 @@ rec {
   mkMonitor = { name, type, query, message ? pagerDutyPolicy.normal, monitorOptions ? {} }:
     { config, ... }:
     baseMonitor // {
-      inherit name type message;
+      inherit type message;
+      name = "${config.deployment.name}: ${name}";
```

Comment:
Does this belong to a separate PR?

Reply:
This fixes evaluation, which was broken previously due to CI failing.

Reply:
Understood, thanks!